daemon/containerd/image.go

```diff
@@ -122,6 +122,36 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 	return img, nil
 }
 
+func (i *ImageService) GetImageVariant(ctx context.Context, refOrID string, options imagetype.GetImageOpts) (*ocispec.Descriptor, error) {
```

Comment:
As discussed a  little bit elsewhere in this PR, I think "variant" is a _slightly_ confusing choice for this function since that's used in OCI for the "v7" part of a platform value like `linux/arm/v7` (`{"os":"linux","architecture":"arm","variant":"v7"}`), but I don't think this is _too_ sticky of a point as long as we don't write it to disk anywhere or encode it in an API we can't change or drop later. :sweat_smile:

Maybe get `GetPlatformImage` or something like that to make it clear this is to take a potentially-index/list ref and return a specific descriptor for a "resolved" single platform image instead?

Reply:
I guess it could even be `GetImageManifest` (matching @thaJeztah's suggestion for the related field name) :eyes:

Reply:
`GetImageManifest` or `ResolveImageManifest` works for me (naming is hard!)

Reply:
I like `GetImageManifest` following the principle of symmetry with the 'legacy'/less specific counterpart (e.g. `Image`, `ImageManifest`).

Reply:
Sounds great! :)